uk.co.compucorp.civicrm.hrleaveandabsences/tests/phpunit/bootstrap.php

```diff
@@ -1,5 +1,5 @@
 <?php
-
+define('CIVICRM_CONTAINER_CACHE', 'never');
```

Comment:
We should probably have the define statements grouped together. Maybe move the `define('CIVICRM_MAILER_SPOOL', 1);` up also to after this line